content/en/blog/2024/year-in-review.md

```diff
@@ -0,0 +1,113 @@
+---
+title: '2024: The Year in Review for opentelemetry.io'
+linkTitle: 'Year in Review'
+date: 2024-12-12 # Put the current date, we will keep the date updated until your PR is merged
+author:
+  >- # If you have only one author, then add the single name on this line in quotes.
+  [Severin Neumann](https://github.com/svrnm) (Cisco),
+sig: SIG Comms
+---
+
+As 2024 comes to a close, we'd like to take the time to take a look back at it,
+and share some insights and accomplishments of SIG Communcation, which is
+responsible for running this website, blog and documentation.
+
+## Contributions
+
+In
+[December 2022](https://github.com/open-telemetry/opentelemetry.io/releases/tag/2022.12)
+we started publishing monthly releases of our website on GitHub, to have a
+regular summary of contributions. Based on this data we can do a long time
+comparison on contributions, and if we look at the time between
+[December, 2022 and November, 2023](https://github.com/open-telemetry/opentelemetry.io/compare/2022.12...2023.11)
+and compare it with
+[December, 2023 to November, 2024](https://github.com/open-telemetry/opentelemetry.io/compare/2023.12...2024.11),
+we see an upwards trend in commits from 1,011 to 1,340, and contributors from 92
+to 106. The only metric which has gone down is the number of files changed,
+which was 1,864 previously and went down to 1,624.
+
+Overall there have been 768 contributors and 3,982 commits across 3,824 merged
+pull requests since the repository was created in April, 2019.
+
+We thank every contributor for helping to build and improve the OpenTelemetry
+website!
```

Comment:
```suggestion
In
[December 2022](https://github.com/open-telemetry/opentelemetry.io/releases/tag/2022.12),
we began publishing monthly releases of our website on GitHub to provide regular
summaries of contributions. Using this data, we can perform long-term
comparisons of contributions. For example, when comparing the period from
[December 2022 to November 2023](https://github.com/open-telemetry/opentelemetry.io/compare/2022.12...2023.11)
with
[December 2023 to November 2024](https://github.com/open-telemetry/opentelemetry.io/compare/2023.12...2024.11),
we see an upward trend: commits increased from 1,011 to 1,340, and contributors
grew from 92 to 106. The only metric that decreased was the number of files
changed, which dropped from 1,864 to 1,624.

Since the repository's creation in April 2019, there have been 768 contributors
and 3,982 commits across 3,824 merged pull requests.

We extend our heartfelt thanks to every contributor for helping build and
improve the OpenTelemetry website!
```